content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/250-querying-the-database.mdx

```diff
@@ -386,6 +386,32 @@ Fantastic, you just wrote new data into your database for the first time using P
 
 </SwitchTech>
 
+<SwitchTech technologies={['node', 'planetscale']}>
```

Comment:
This page is missing the tech dBswitcher in the frontmatter for this to work - currently there is no `planetscale` variant of this page.

Reply:
Done